meilisearch-http/public/interface.html

```diff
@@ -169,7 +177,8 @@ <h2 class="subtitle">
         lastRequest.open("GET", theUrl, true);
         lastRequest.onload = function (e) {
             if (lastRequest.readyState === 4 && lastRequest.status === 200) {
-                let httpResults = JSON.parse(lastRequest.responseText);
+                let sanitizedResponseText = lastRequest.responseText;
+                let httpResults = JSON.parse(sanitizedResponseText);
```

Comment:
Whooops... Don't you forget to call your sanitize function, here?

Reply:
Duh! I'm an idiot. Good catch. I think I gave you a wrong version of my changes :( I did it quickly before joining a meeting. Feel free to call the function. Or I can change it after my endless meetings. :)

Reply:
Take your time, change it when you have time, no problem :)